app.js

```diff
@@ -37,7 +37,7 @@ app.use(passport.session());
 // Use CSRF Protection
 // TODO: Add PUT here is just for debug
 // TODO: Add csrf token to the profile.handlebars
-app.use(csrf({ignoreMethods: ['PUT', 'GET', 'HEAD', 'OPTIONS']}));
+app.use(csrf({ignoreMethods: ['POST','PUT', 'GET', 'HEAD', 'OPTIONS']}));
```

Comment:
This is just for debug. We probably don't need this in production. We want the CSRF extension to work.